toy_store/demo/website_view.xml

```diff
@@ -0,0 +1,232 @@
+                                            <br/>
+                                        </p>
+                                        <p class="mb-0">
+                                            <a class="btn btn-primary btn-lg" style="border-radius:0.6rem;" href="/shop" data-bs-original-title="" title="">Shop <span class="fa fa-angle-right ms-2"/></a>
+                                        </p>
+                                    </div>
+                                    <div class="o_grid_item o_grid_item_image o_colored_level g-col-lg-3 g-height-7 col-lg-3" style="z-index: 2; grid-area: 1 / 9 / 8 / 12;">
+                                        <img src="/web/image/toy_store.ir_attachment_1009" alt="" class="img img-fluid mx-auto o_we_custom_image" data-mimetype="image/webp" data-original-id="1007" data-original-src="/web/image/1007-f0631c4e/81hzE-O0hxL._SL1500_.png" data-mimetype-before-conversion="image/png" data-resize-width="NaN" loading="lazy"/>
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                        <section class="s_title pt40 pb40 o_colored_level" data-vcss="001" data-snippet="s_title" data-name="Title">
+                            <div class="s_allow_columns container">
+                                <h2 class="display-3-fs o_default_snippet_text" style="text-align: center;">Our <span class="o_text_highlight o_text_highlight_freehand_3 o_text_highlight_fill" style="--text-highlight-width: 6px; --text-highlight-color:#D15C01;">selection</span></h2>
+                            </div>
+                        </section>
+                        <section data-snippet="s_dynamic_snippet_products" class="s_dynamic_snippet_products s_dynamic pt32 pb32 o_colored_level s_product_product_centered o_dynamic_empty" data-name="Products" data-product-category-id="all" data-show-variants="" data-number-of-records="16" data-filter-id="2" data-template-key="website_sale.dynamic_filter_template_product_product_centered" data-carousel-interval="5000" data-product-tag-ids="[{&quot;id&quot;:1,&quot;name&quot;:&quot;Featured&quot;,&quot;display_name&quot;:&quot;Featured&quot;}]" data-arrow-position="bottom">
+                            <div class="o_not_editable container">
+                                <div class="css_non_editable_mode_hidden">
+                                    <div class="missing_option_warning alert alert-info rounded-0 fade show d-none d-print-none o_default_snippet_text">
+                                        Your Dynamic Snippet will be displayed here... This message is displayed because you did not provided both a filter and a template to use.<br/>
+                                    </div>
+                                </div>
+                                <div class="dynamic_snippet_template"/>
+                            </div>
+                        </section>
```

Comment:
```suggestion
                        <section data-snippet="s_dynamic_snippet_products" class="s_dynamic_snippet_products s_dynamic pt32 pb32 o_colored_level s_product_product_centered o_dynamic_empty" data-name="Products" data-product-category-id="all" data-show-variants="" data-number-of-records="16" data-filter-id="2" data-template-key="website_sale.dynamic_filter_template_product_product_centered" data-carousel-interval="5000" data-product-tag-ids="[{&quot;id&quot;:1,&quot;name&quot;:&quot;Featured&quot;,&quot;display_name&quot;:&quot;Featured&quot;}]" data-arrow-position="bottom">
                            <div class="o_not_editable container">
                                <div class="css_non_editable_mode_hidden">
                                    <div class="missing_option_warning alert alert-info rounded-0 fade show d-none d-print-none o_default_snippet_text">
                                        Your Dynamic Snippet will be displayed here... This message is displayed because you did not provided both a filter and a template to use.<br/>
                                    </div>
                                </div>
                                <div class="dynamic_snippet_template"/>
                            </div>
                        </section>
```
```suggestion
                        <section data-snippet="s_dynamic_snippet_products" class="s_dynamic_snippet_products s_dynamic pt32 pb32 o_colored_level s_product_product_centered o_dynamic_empty" data-name="Products" data-product-category-id="all" data-show-variants="true" data-number-of-records="16" data-filter-id="1" data-template-key="website_sale.dynamic_filter_template_product_product_centered" data-carousel-interval="5000" data-arrow-position="bottom">
                            <div class="container o_not_editable">
                                <div class="css_non_editable_mode_hidden">
                                    <div class="missing_option_warning alert alert-info rounded-0 fade show d-none d-print-none o_default_snippet_text">
                                        Your Dynamic Snippet will be displayed here... This message is displayed because you did not provided both a filter and a template to use.<br/>
                                    </div>
                                </div>
                                <div class="dynamic_snippet_template"/>
                            </div>
                        </section>
```